test/e2e/flask/btc/btc-account-overview.spec.ts

```diff
@@ -5,7 +5,7 @@ import BitcoinHomepage from '../../page-objects/pages/home/bitcoin-homepage';
 import { withBtcAccountSnap } from './common-btc';
 
 describe('BTC Account - Overview', function (this: Suite) {
-  it('has balance displayed and has portfolio button enabled for BTC accounts', async function () {
+  it.skip('has balance displayed and has portfolio button enabled for BTC accounts', async function () {
```

Comment:
maybe i miss some context here , but can you help me to understand why we are not doing something like:
```
    if (!getIsBitcoinSupportEnabled(state)) {
      return;
    }
    ```
rather than skip, on the e2e tests ?

Reply:
We cannot use those kind of selector in e2e tests. And most of the Bitcoin e2e test are actually mocking "enabling" this flag with some fixtures.

Like here: https://github.com/MetaMask/metamask-extension/blob/main/test/e2e/flask/btc/common-btc.ts#L223

Also, we cannot use code-fence in e2e tests (at least, to my knowledge), so we would need to find a trick (using a test build)

Unless there are some known tricks to do this?